app/forms/webauthn_visit_form.rb

```diff
@@ -41,10 +41,9 @@ def check_params(params)
     return unless error
 
     if @platform_authenticator
-      errors.add error, translate_platform_authenticator_error(error),
-                 type: :"#{translate_platform_authenticator_error(error).split('.').last}"
+      errors.add error, :invalid, message: translate_platform_authenticator_error(error)
     else
-      errors.add error, translate_error(error), type: :"#{translate_error(error).split('.').last}"
+      errors.add error, :invalid, message: translate_error(error)
```

Comment:
Noting that this is a change in behavior, though the previous implementation would log `error_details` with string keys derived from the user-facing string, which would make querying much more difficult. As best I can tell, we don't have any existing dashboards or alarms which relied on the previous format.

Reply:
Granting that this is still not a consistent format even after the changes, since keys of `error_details` should refer to the specific problematic field, not the error itself. This page is a bit unique in how it logs errors, but a better format might be something like...

```
error_messages: { client_data_json: { invalid_state: true } }
```

`client_data_json` referring to [one of the fields](https://github.com/18F/identity-idp/blob/4eddf187da7b26842931d1b6df0ba89727fbabec/app/javascript/packs/webauthn-setup.ts#L62-L73) expected to be populated in the client-side form.